.github/workflows/meson.yml

```diff
@@ -15,105 +15,102 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {
-            name: Windows MSVC Release,
-            os: windows-latest,
-            msvc: true,
-            msystem: MINGW64,
-            shell: 'powershell',
-            buildtype: release,
+          - name: Windows (MSVC)
+            os: windows-latest
+            msvc: true
+            shell: 'pwsh'
             args: '-Ddefault_library=both'
-          }
-          - {
-            name: Windows MinGW Release,
-            os: windows-latest,
-            msvc: false,
-            pkgmanager: msys,
-            msystem: MINGW64,
-            shell: 'msys2 {0}',
-            buildtype: debugoptimized,
+            devenv: |
+              Import-Module "$env:VS\Common7\Tools\Microsoft.VisualStudio.DevShell.dll"
+              Enter-VsDevShell -VsInstallPath $env:VS -SkipAutomaticLocation -DevCmdArguments "-arch=x64 -host_arch=x64"
+
+          - name: Windows (CLANG64)
+            os: windows-latest
+            msystem: CLANG64
+            shell: 'msys2 {0}'
+            args: '-Ddefault_library=both -Db_sanitize=address,undefined'
+
+          - name: Windows (UCRT64)
+            os: windows-latest
+            msystem: UCRT64
+            shell: 'msys2 {0}'
             args: '-Ddefault_library=both'
```

Comment:
Might be good to use `-Ddefault_library=shared` for one Windows runner to ensure shared-only functions too

Reply:
Sure, I can make each of this build different.